homeassistant/components/recorder/__init__.py

```diff
@@ -138,13 +142,23 @@ def run_information(hass, point_in_time: Optional[datetime] = None):
     if not db_url:
         db_url = DEFAULT_URL.format(hass_config_path=hass.config.path(DEFAULT_DB_FILE))
 
+    db_max_retries = conf.get(CONF_DB_MAX_RETRIES, None)
```

Comment:
Use `dict[key]` for optional config keys that also have a default value set.

Reply:
I believe i did what youre asking in 8bac36bdd014bd9f2f5299aae40019bf23ea6ad7 - I'm not entirely sure. If you could review my solution, that would be great :-)

Reply:
Yes, thank you very much 👍  just one thing needs to be solved, so please take a look at my suggestion :)

Reply:
Ah! Gotcha. I've updated __init__.py with the changes you suggested in d11d446a7b42f8c21c8fe09329caa1b488109a12